org/ystia/topologies/a4c_yorc_basic/types.yml

```diff
@@ -217,7 +217,7 @@ topology_template:
     TerraformRuntime:
       type: org.ystia.terraform.linux.terraform.nodes.TerraformRuntime
       properties:
-        component_version: "0.9.11"
+        component_version: "0.11.8"
```

Comment:
We should also update the default version of the TerraformRuntime component in org/ystia/terraform ?

Reply:
Yes, I miss it